registrations/tests/test_registration_admin_side.py

```diff
@@ -54,6 +54,21 @@ def test_create_registration(api_client, event, has_right_to_edit, expected_stat
     assert response.status_code == expected_status
 
 
+@pytest.mark.parametrize(
+    "event_type",
+    [Event.TypeId.GENERAL, Event.TypeId.COURSE, Event.TypeId.VOLUNTEERING],
+)
+@pytest.mark.django_db
+def test_get_registration(api_client, event, event_type, registration):
+    event.type_id = event_type
+    event.save()
+
+    registration_url = reverse("registration-detail", kwargs={"pk": registration.id})
+    response = api_client.get(registration_url)
+
+    assert response.status_code == status.HTTP_200_OK
```

Comment:
Check that the content is somewhat sane as well. Or at least in my experience, 200 OK doesn't always mean it's actually OK.
```suggestion
    assert response.status_code == status.HTTP_200_OK
    assert response.data["id"] == registration.id
```

Reply:
Totally agree. Added an assertion for registration id